pbiviz.json

```diff
@@ -2,14 +2,14 @@
   "visual": {
     "name": "drilldownChoropleth",
     "displayName": "Drilldown choropleth",
-    "guid": "choropleth4C77906D867C4C09AFE83BA5E2AA106A",
+    "guid": "choropleth4C77906D867C4C09AFE83BA5E2AA105A",
```

Comment:
GUID shouldn't be changed if we want to deliver the new version to all of end users